mars/PCR.lfr

```diff
@@ -0,0 +1,29 @@
+module PCR(
+    finput A,
+    foutput a_out,
+    control c1, c2
+);
+
+    flow mixer_in, mixed, incubated;
+    #MAP 'MIX' '+'
+    #MAP 'Incubator' '~'
```

Comment:
The `#MAP` only apply for the immediate next assign statement

Reply:
They are used in the following distribute block, and declaring MAP in the block causes an error. (please refer to my reply to the ligation.lfr)